.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,38 @@
+name: golangci_lint
+
+on:
+  push:
+
+jobs:
+  golang_lint:
+    name: Golang Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@f25a3a9f25bd5f4c5d77189cab02ff357b5aedeb # v2.4.1
+      - uses: smartcontractkit/tool-versions-to-env-action@v1.0.7
+        id: tool-versions
+      - name: Setup go ${{ steps.tool-versions.outputs.golang_version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.tool-versions.outputs.golang_version }}
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: v${{ steps.tool-versions.outputs.golangci-lint_version }}
+
+          # Optional: working directory, useful for monorepos
+          working-directory: relayer
+
+          # Optional: golangci-lint command line arguments.
+          args: --enable=gofmt --tests=false --exclude-use-default --timeout=5m0s
```

Could drop this CI timeout since the `.golangci.yml` specifies one now.
```suggestion
          args: --enable=gofmt --tests=false --exclude-use-default
```